templates/lab-results/README.md

```diff
@@ -0,0 +1,9 @@
+Sample lab result document
```

Comment:
I see that you are sometimes using `lab results`, sometimes `lab result` (singular) when you are referring to the example. Please choose one of them and stick with that. There are multiple occurrences of both throughout the code and `README.md` files.